File: python/6 kyu/275.py
# https://www.codewars.com/kata/5727bb0fe81185ae62000ae3/train/python

# Passed

def clean_string(s):
    modified_characters = []
    for char in s:
        if char == "#":
            if len(modified_characters) == 0:
                continue

            modified_characters.pop()
        else:
            modified_characters.append(char)

    result = "".join(modified_characters)
    return result

output = clean_string('abc#d##c')
print(output)

File: nodejs/7 kyu/275.js
// https://www.codewars.com/kata/56b97b776ffcea598a0006f2/train/javascript

// Passed

function bubblesortOnce(a) {
  let arr = [...a];
  for (let idx = 0; idx < arr.length - 1; idx++) {
    if (arr[idx] > arr[idx + 1]) {
      [arr[idx], arr[idx + 1]] = [arr[idx + 1], arr[idx]];
    }
  }

  return arr;
}

const output = bubblesortOnce([9, 7, 5, 3, 1, 2, 4, 6, 8]);
console.log(output);